### PR TITLE
Fixes #490 - Redesign Home page of server

### LINKED ISSUE #490: Redesign start page of server similar to other services
* Implement top bar same as web chat including the menu dropdown on the right
* Add footer same as on web chat overview page
* Change the following
   * "Software and Rules for Personal Assistants" -> "SUSI.AI API Server for Artificial Intelligence Personal Assistants, Robots, Help Desks and Chatbots"
   * "Chat with a Personal Assistant: Artificial Intelligence for Information Retrieval." -> "This server provides an output of intelligent answers in JSON format for SUSI.AI clients" 
* Change font and style to match web chat service
* Add .ico same as on other pages
* Take out "Nice? To get latest news about susi, follow @0rb1t3r. Help to spread the news: Tweet about this"

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes #490 

Changes: 

- Changed  colour scheme  and fonts 
- Added Footer
- Contend Replacements 

Screenshots for the change: 
screen shot 2017-08-17 at 7 03 19 am
screen shot 2017-08-20 at 9 31 00 am

screen shot 2017-08-17 at 8 18 37 am
screen shot 2017-08-20 at 9 31 17 am

@uday96  @rishiraj824  @madhavrathi  @dynamitechetan  @DravitLochan  please review this

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -35,74 +46,213 @@
...
-            <br/><span style="font-size:7pt;">
+      <br>
+      <br>
+      <form action="#" class="col-lg-12 hidden-xs">
+        <input id="timezoneOffset0" type="hidden" name="timezoneOffset" value="">
+        <script>
+          document.getElementById("timezoneOffset0").value = new Date().getTimezoneOffset();
+        </script>
+        <div class="input-group input-group-lg col-sm-8" style="float:left">
+          <input name="q" id="query" type="text" class="center-block form-control input-lg" title="Search Twitter and the loklak network"
```
Comment (betterclever) [inline html/index.html]: @isuruAb Please check this once.

File: html/index.html [hunk 2]
```diff
@@ -35,74 +46,213 @@
...
+        </div>
+      </form>
+
+      <form action="#" class="col-lg-12 hidden-lg hidden-md hidden-sm row">
+        <input id="timezoneOffset1" type="hidden" name="timezoneOffset" value="">
+        <script>
+          document.getElementById("timezoneOffset1").value = new Date().getTimezoneOffset();
+        </script>
+        <div class="input-group input-group-lg row col-sm-12" style="float:left;display: flex;flex-wrap: wrap; padding:10px;">
+          <input name="q" id="query" type="text" class="center-block form-control input-lg" title="Search Twitter and the loklak network"
```
Comment (betterclever) [inline html/index.html]: This one also.

**Comment by isuruAb** [comment]
@betterclever  @rishiraj824  please check this again

File: html/css/loklak.css [hunk 3]
```diff
@@ -1,23 +1,156 @@
 /* Move down content because we have a fixed navbar that is 50px tall */
```
Comment (dynamitechetan) [inline html/css/loklak.css]: this file name is still loklak.css. can it be changed to susi.css

**Comment by royjesslong** [comment]
Good

**Comment by isuruAb** [comment]
@dynamitechetan  @DravitLochan  @rishiraj824  please review

**Comment by mariobehling** [comment]
@isuruAb Let's follow up on this. Looks good to me. Please resolve merge conflicts.

**Comment by isuruAb** [comment]
@mariobehling  please check again

**Pull request merged**